### ISSUE TITLE
🐛 No easy way to stop wunderctl up

### ISSUE THREAD

**Issue by hinogi** [wundergraph/wundergraph#35]
### Bug description

So while there is something like `wunderctl up --debug` that shows debug information. If you exit via ctrl+c the process remains active.
There is no `wunderctl down` so you can't make it stop unless you either memorized the PID or you look it up on which process is using port 9991 or 9992 and kill it manually.

### How to reproduce

1. wunderctl up --debug

### Expected behavior

Make wunderctl down

### WunderGraph information

https://wundergraph.com/docs/guides/your_first_wundergraph_application/overview

### Environment & setup

- OS: Windows 10 10.0.19044
- Node.js version: 16.13.1

### WunderCtl Version

```
Version: 0.90.32
Commit: 386a3d4b66204411dc89da6919aad0bebe524657
Date: 2022-06-06T12:22:22Z
BuiltBy: ci
```

**Comment by jensneuse** [comment]
I think this is already fixed with #33. @StarpTech did we create a new release already?

**Comment by StarpTech** [comment]
But I think this won't solve the issue. `wunderctl up` doesn't call this place. https://github.com/wundergraph/wundergraph/pull/32 might fix it.

**Comment by StarpTech** [comment]
Hi @hinogi, could you give the latest release a shot?

**Comment by hinogi** [comment]
```
2022-06-11T10:37:27+02:00       debug   listener closed {"addr": "127.0.0.1:9991"}
2022-06-11T10:37:27+02:00       debug   listener closed {"addr": "[::1]:9991"}
```
Seems to work now
